src/extension/tsconfigHelper.ts

```diff
@@ -23,24 +24,27 @@ export class TsConfigHelper {
         return fileSystem.exists(tsConfigPath)
             .then(function(exists: boolean): Q.Promise<void> {
                 if (!exists) {
-                    return fileSystem.writeFile(tsConfigPath, "{}");
+                    return fileSystem.writeFile(tsConfigPath, "{ compilerOptions: { allowJs: true } }");
                 }
             })
             .then(function(): Q.Promise<string> {
                 return fileSystem.readFile(tsConfigPath, "utf-8");
             })
             .then(function(jsonContents: string): Q.Promise<any> {
-                return JSON.parse(jsonContents);
+                return JSON.parse(stripJsonComments(jsonContents));
```

Comment:
@MSLaguana if no `stripJsonComments` package, how to parse JSON file with comments? Just using JSON.parse a json file with comments will trigger error.